llvm/test/Transforms/LoopExtractionAnalysis/complex.ll

```diff
@@ -0,0 +1,239 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; REQUIRES: asserts
+; RUN: opt -passes=loop-extract-analysis -pass-remarks-analysis=loop-extract-analysis -debug-only=loop-extract-analysis -S < %s 2>&1 | FileCheck %s
+
+; ModuleID = 'loop-extraction/complex.cpp'
+source_filename = "loop-extraction/complex.cpp"
+target datalayout = "e-m:o-i64:64-i128:128-n32:64-S128"
+target triple = "arm64-apple-macosx12.0.0"
```

Comment:
I think the triple and attributes below shouldn't be needed for the extraction test

Reply:
removed.

Reply:
Also removed metadata and attributes at the bottom.